pkgs/development/python-modules/debugpy/default.nix

```diff
@@ -59,6 +59,7 @@ buildPythonPackage rec {
     ${stdenv.cc}/bin/c++ linux_and_mac/attach.cpp -Ilinux_and_mac -fPIC -nostartfiles ${{
       "x86_64-linux"   = "-shared -m64 -o attach_linux_amd64.so";
       "i686-linux"     = "-shared -m32 -o attach_linux_x86.so";
+      "aarch64-linux"  = "-shared -o attach_linux_arm64.so";
```

Comment:
I based these arguments off of `linux_and_mac/compile_linux.sh` & `linux_and_mac/compile_mac.sh`, but I suppose we don't actually need to specify the architecture with the way stdenv is setup :sweat_smile:

Reply:
I would be happy to add them for consistency if you prefer, though! The valid options for `-march` when I run it under qemu (via `nix-build -A python39Packages.debugpy --argstr system aarch64-linux`, after enabling binfmt emulation with `boot.binfmt.emulatedSystems = [ "aarch64-linux" ];`) are `armv8-a armv8.1-a armv8.2-a armv8.3-a armv8.4-a armv8.5-a native`.

Reply:
Eh, I don't mind :sweat_smile:. I think I would actually prefer to remove them everywhere else, but that's not really necessary in this PR, and can be done in the next update.